include/irr/asset/ICPUImage.h

```diff
@@ -30,10 +30,10 @@ class ICPUImage final : public IImage, public IAsset
         core::smart_refctd_ptr<IAsset> clone(uint32_t _depth = ~0u) const override
         {
             auto par = params;
-            auto cp = core::make_smart_refctd_ptr<ICPUImage>(std::move(par));
+            auto cp = core::smart_refctd_ptr<ICPUImage>(new ICPUImage(std::move(par)), core::dont_grab);
```

Comment:
why @Crisspl ?

Reply:
ctor is inaccessible for make_smart...() func

Reply:
ah ok, then use `create`